Yoast/YoastSEO.js
# Stories/1331 add reading time indicator

## Summary

This PR can be summarized in the following changelog entry: adds a researcher to calculate the reading time for a given paper.

## Relevant technical choices:

* The reading time is always rounded up to the next full minute (e.g. 0.2 -> 1) to provide a conservative measure.
* Currently this assessment uses a basic formula based on text length and the number of images. The original reading time formula (see https://github.com/Yoast/yoast.com/issues/1395) also included code tags. With the current text parsing method we cannot extract this information. This functionality will be added later when we have the relevant text parsing ability.
* The reading time indicator is initially intended for use on yoast.com only. It currently is not meant to be accessed from within the content analysis of the plugin.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` to check whether none of the unit tests fail.
* Create an additional test using a text of your choice. Check whether it returns a meaningful reading time.

Fixes #1331
